ui/src/app/pages/workspace/workspace-edit.spec.tsx

```diff
@@ -216,6 +215,43 @@ describe('WorkspaceEdit', () => {
     expect(navigate).toHaveBeenCalledTimes(1);
   });
 
+  it('defaults to upgrading the CDR Version when duplicating a workspace with an older CDR Version', async() => {
+    // init the workspace to a non-default CDR version value
+    const altCdrWorkspace = {...workspace, cdrVersionId: CdrVersionsStubVariables.ALT_WORKSPACE_CDR_VERSION_ID}
+    currentWorkspaceStore.next(altCdrWorkspace);
+
+    // duplication will involve a CDR version upgrade by default
+    routeConfigDataStore.next({mode: WorkspaceEditMode.Duplicate});
+
+    const wrapper = component();
+    await waitOneTickAndUpdate(wrapper);
+
+    const cdrSelection = wrapper.find('[data-test-id="select-cdr-version"]').find('select').props().value;
+
+    // default CDR version, not the existing workspace's alt CDR version
+    expect(cdrSelection).toBe(CdrVersionsStubVariables.DEFAULT_WORKSPACE_CDR_VERSION_ID);
+
+    const expectedUpgradeMessage1 = `You're duplicating the workspace "${altCdrWorkspace.name}" to upgrade from`;
```

Comment:
nit: Same comment as above. The only reason I call it out is to prevent needing to change the test for minor language changes.